app/models/run.rb

```diff
@@ -14,11 +14,28 @@ class Run < ActiveRecord::Base
   before_create :escape_jquery_html_characters
   after_create :add_run_context_to_name
   after_save :update_sendable_status
+  before_save :do_not_update_completed_runs
```

Comment:
This method has not been defined in the controller, or the relevant change is missing from this PR currently.